test/testdata/rewriter/dsl_builder.rb.rewrite-tree.exp

```diff
@@ -1,5 +1,7 @@
 class <emptyTree><<C <root>>> < ()
   class <emptyTree>::<C TestDSLBuilder><<C <todo sym>>> < (::<todo sym>)
+    <self>.dsl_optional(:"no_getter_or_setter", <emptyTree>::<C String>, {:"skip_getter" => true, :"skip_setter" => true})
```

Comment:
why did the order change?

Reply:
This PR makes it so that method definitions and their attached sigs are always moved to the end of the class. This was not a method definition, so it did not get moved: hence why it is now here.